Developer-Guide.md

```diff
@@ -166,6 +188,41 @@ $ sudo install -o root -g root -m 0640 -D kata-containers.img "/usr/share/kata-c
 $ (cd /usr/share/kata-containers && sudo ln -sf "$image" kata-containers.img)
 ```
 
+## Create an initrd image - OPTIONAL
+### Create a local rootfs for initrd image
+```
+$ cd $GOPATH/src/github.com/kata-containers/osbuilder/rootfs-builder
+$ script -fec 'sudo -E GOPATH=$GOPATH AGENT_INIT=yes USE_DOCKER=true ./rootfs.sh ${distro}'
+```
+`AGENT_INIT` controls if the guest image will use kata agent as the guest `init` process. When creating an
+initrd image, we always want to enable it.
+
+`${distro}` can be one of `alpine`, `centos`, `clearlinux`, `euleros` and `fedora`. When unset, it builds
```

Comment:
Lines 200-201 suggested rewrite: You can choose alpine, centos, clearlinux, euleros, or fedora for ${distro}. If ${distro} is not set, it builds 'centos' based rootfs by default.